### PR TITLE
Added Concourse build plan for EQ

### PR DESCRIPTION
**Pull request by jonnyshaw89**
This PR adds the concourse build plan for EQ along with the build containers required to run it.

### CODE REVIEW

**Comment by ajmaddaford** [comment]
I think we need to look at splitting up the concourse pipeline yaml to make it more readable. 

Are there options around separating out build, deploy, test jobs? Also, some other projects separate tasks out into individual files e.g. https://github.com/cloudfoundry/buildpacks-ci/blob/master/pipelines/sample-apps.yml.

We should remove all blank lines within each job yaml to make the distinction between jobs clearer.

**Comment by jonnyshaw89** [comment]
@ajmaddaford With regards to splitting the tasks into separate files, the individual files would either need to be put into the repo that they relate to (eg the SR build and deploy tasks going into the eq-survey-runner repo) or the eq-deploy repo would need to be checked out by the jobs when they run.

Which of those options do you prefer? also there is the option to do neither yet.

**Comment by ajmaddaford** [comment]
I'd prefer for the build and test tasks to live with the services they relate to. As this will need to be done for each individual service I think we should merge this as is and create cards to do this for each service.

**Review by ajmaddaford — APPROVED** [review]

File: eq.yml [hunk 1]
```diff
@@ -0,0 +1,810 @@
...
+  - put: eq-schema-validator-image
+    params:
+      build: compiled-eq-schema-validator
+      tag: eq-schema-validator/.git/HEAD
+    get_params:
+      skip_download: true
+
+- name: staging-deploy
+  serial_groups: [staging-deploy, staging-functional-tests-core, staging-functional-tests-ukis, staging-functional-tests-census, staging-destroy]
+  plan:
```
Comment (shahi645) [inline eq.yml:396]: @jonnyshaw89 dont we need to add `eq-smoke-test` here?

Comment (jonnyshaw89) [reply eq.yml:396]: Not until its merged. that can go in as a later PR

**Review by shahi645 — APPROVED** [review]
@jonnyshaw89 squishy-squashy

**Pull request merged**